config/webpack/rules/node_modules.js

```diff
@@ -2,7 +2,7 @@ const { join } = require('path');
 const { settings, env } = require('../configuration');
 
 module.exports = {
-  test: /\.(js|mjs)$/,
```

Comment:
I think add ts/tsx to test of node_modules rule are not needed (since dependencies packages should be already transpiled to JS).